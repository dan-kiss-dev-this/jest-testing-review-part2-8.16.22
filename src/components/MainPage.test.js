import { shallow } from 'enzyme';
import React from 'react';
import App from '../containers/App.js';

it('should render the app component', () => {
  const mockStore = {
    robots: [],
    searchField: '',
  };
  expect(shallow(<App store={mockStore} />)).toMatchSnapshot();
});
