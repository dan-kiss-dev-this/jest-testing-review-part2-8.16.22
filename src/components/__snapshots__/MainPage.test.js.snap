// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the MainPage component without crashing 1`] = `
<div
  className="tc"
>
  <Header />
  <SearchBox />
  <Scroll>
    <ErrorBoundary>
      <CardList
        robots={Array []}
      />
    </ErrorBoundary>
  </Scroll>
</div>
`;
